src/translations/fr.json

```diff
@@ -18,6 +18,7 @@
   "components.ConnectWalletModal.components.LedgerChecklist.more": "",
   "components.ConnectWalletModal.components.LedgerChecklist.secondCheck": "",
   "components.ConnectWalletModal.components.LedgerChecklist.thirdCheck": "",
+  "components.ConnectWalletModal.components.SelectPreferredNetwork.forkNetwork": "",
```

Comment:
@sakulstra do we manaully add all these to every single language file?

Reply:
no it's generated via this extract utility https://github.com/aave/aave-ui/pull/225/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R121 

This was usually executed via husky on precommit, but was broken for some time without someone noticing. This is why now when it's fixed these changes are in the pr.